nova/src/provider/zeromorph.rs

```diff
@@ -91,7 +94,21 @@ where
         )
         .entered();
 
-        SpartanZM::setup(max_poly_vars, label, rng)
```

Comment:
Why is this change needed?

In practice it's not going to be used as SRS requires trusted setup.

Reply:
Just trying to make test setup as low touch as possible

Reply:
I guess I can push this elsewhere though, will change shortly

Reply:
Okay I mean why does this code **need** to be somewhere.
This kind of matters for Pedersen bases, for tests you can use test_rng and it's good enough without additional hashing and seeds.

Reply:
I guess that's true, okay I can strip it down to the `test_rng`.